### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by soulspark666**
Readme.MD Updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
-# Zodiac Signs
\ No newline at end of file
+# Zodiac Signs
+Zodiac Signs is a web app developed in React. This web appcan be used to check your zodiac signs.
```
Comment (esslamben) [inline README.md]: spelling mistake on appcan here mate :)

File: README.md [hunk 2]
```diff
@@ -1 +1,14 @@
...
+
+## Usage
+Open [this](https://dipanshkhandelwal.github.io/my-zodiac-sign/) link.
+Enter your birhtday.
+Voila you know your Zodiac sign.
+
+## Screenshots
+![Main Page](https://raw.githubusercontent.com/soulspark666/my-zodiac-sign/master/Screenshots/Page.png)
+
+![Srorpio](https://raw.githubusercontent.com/soulspark666/my-zodiac-sign/master/Screenshots/Scorpio.png)
```
Comment (esslamben) [inline README.md]: Srorpio -> Scorpio

File: README.md (hunk 1 above)
Comment (soulspark666) [reply README.md]: Sorry didn't see it!

File: README.md (hunk 2 above)
Comment (soulspark666) [reply README.md]: Sorry didn't see it! Good project by the way

**Comment by soulspark666** [comment]
Edited the mistakes.

**Comment by DipanshKhandelwal** [comment]
Thanks but its already done. Please take up something else.

**Pull request closed without merging**